tests/unit/tasks/server/express-server-test.js

```diff
@@ -692,7 +692,7 @@ describe('express-server', function() {
 
       it('calls serverMiddleware on the addons on start', function() {
         return subject.start({
-          host:  '0.0.0.0',
+          host: undefined,
```

Comment:
I left it in, because it's more explicit. Might make it easier to quickly scan the tests.

But I can remove it and squash the commits, if you want. :)